### ISSUE TITLE
Update syntax from CJS to ESM

### ISSUE THREAD

**Issue by uncenter** [uncenter/learn-eleventy#7]

**Comment by SteelMaverick** [comment]
I have a suggestion for approaching this. Since the web was built with backwards compatibility in mind, perhaps there should be a way to display both ESM and CJS implementations using like a tablist. Similar to what Eleventy's site does for showing various types of code snippets.

**Comment by uncenter** [comment]
I don't think that makes sense for Learn Eleventy in particular. This is meant to be a tutorial, and we should push users toward modern standards like ESM instead of even offering older stuff like CJS. I also don't want to have to maintain the burden of basically two sets of code.
